src/internal/extenders/withExtras.ts

```diff
@@ -0,0 +1,14 @@
+import {AnalyticsContext} from '../AnalyticsContext';
+import {DataMap} from '../types';
+import {identity, isObject} from './utils';
+import {withContext} from './withContext';
+
+export function withExtras(extras: DataMap) {
+  if (!isObject(extras)) {
```

Comment:
This seems like an error for sure, it's worth writing a log about it (in development mode)
Same for `withIdentities`/`withFilters`

Reply:
that's the previous behavior. if it were up to me I would throw an exception here